pkg/scheduler/placement/fixed_rule_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/apache/incubator-yunikorn-core/pkg/cache"
 	"github.com/apache/incubator-yunikorn-core/pkg/common/configs"
 	"github.com/apache/incubator-yunikorn-core/pkg/common/security"
+	"gotest.tools/assert"
```

Comment:
Please run make lint over the change I think it will fail on ordering of the imports
An other tool you have is to just run goimports locally to see what it complains about: `goimports -d -local "github.com/apache/incubator-yunikorn" .`
This same comment applies to lots of import changes in this PR

Reply:
I still got `-bash: goimports: command not found` but fixed it manually.